### PR TITLE
Pull riscv64 sources from default repository

### PR DESCRIPTION
**Pull request by luhenry**
Since https://openjdk.org/jeps/422 has been merged in JDK 19, we don't need to pull from the riscv-port repo anymore and we can pull from the default repo instead.

### CODE REVIEW

**Review by karianna — APPROVED** [review]
LGTM - holding off merging until after release week though.

**Comment by karianna** [comment]
@luhenry Are you also able to sign the Eclipse CLA?

PS:  Nice to see you here!

**Comment by luhenry** [comment]
@karianna I'm working on signing the Eclipse CLA. I'll come back to this PR as soon as it's done.

**Comment by gdams** [comment]
@luhenry here's the docs if you need help https://adoptium.net/docs/eca-sign-off

File: pipelines/jobs/configurations/jdk19_pipeline_config.groovy
Comment (gdams) [inline pipelines/jobs/configurations/jdk19_pipeline_config.groovy:173]: do you know if we still need to pass `--with-native-debug-symbols=none` to the configure command? That doesn't seem quite right

Comment (luhenry) [reply pipelines/jobs/configurations/jdk19_pipeline_config.groovy:173]: We don’t in upstream OpenJDK GHA. Not sure why it’s here in the first place though. Also, getting the debug symbols is very handy.

Comment (gdams) [reply pipelines/jobs/configurations/jdk19_pipeline_config.groovy:173]: ping @sxa who likey has more insight here

**Comment by luhenry** [comment]
I was able to sign the ECA. It should be now good to review.

File: pipelines/jobs/configurations/jdk19_pipeline_config.groovy
Comment (luhenry) [reply pipelines/jobs/configurations/jdk19_pipeline_config.groovy]: From discussion on Slack, it was disabled due to the extra cost of storing/transferring the data in and out of the bots. Let's reenable it and disable it if/when it breaks.

**Review by sxa — COMMENTED** [review]
LGTM - thanks. I've let the workflows run now so if they pass I'll approve and merge

File: pipelines/jobs/configurations/jdk19_pipeline_config.groovy
Comment (sxa) [reply pipelines/jobs/configurations/jdk19_pipeline_config.groovy]: Now changed - resolving

**Review by sxa — APPROVED** [review]

**Pull request merged**
